modules/identify-vulnerabilities-in-nodes-vm20.adoc

```diff
@@ -41,7 +41,10 @@ a|
 a|
 * *Name*: The name of the CVE.
 * *Discovered time*: The date when {product-title-short} discovered the CVE.
-* *CVSS*: The severity level for the CVE. You can select from the following options for the severity level:
+* *CVSS*: The severity level for the CVE. 
```

Comment:
Consider adding "Common Vulnerability Scoring System (CVSS)" to explain the acronym.

See https://access.redhat.com/security/updates/classification

Reply:
I would prefer to leave it as it is because we are explaining about the UI elements.